ccd-definition/CaseEvent.json

```diff
@@ -283,20 +283,6 @@
     "ShowEventNotes": "N",
     "EndButtonLabel": "Delete application"
   },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "uploadDocumentsAfterSubmission",
-    "Name": "Documents",
-    "Description": "Upload documents after submission",
-    "DisplayOrder": 1,
-    "PreConditionState(s)": "Submitted",
-    "PostConditionState": "Submitted",
-    "SecurityClassification": "Public",
-    "ShowSummary": "N",
-    "ShowEventNotes": "N",
-    "EndButtonLabel": "Save and continue"
```

Comment:
@swalker125 - do you think can those events be removed? 

What will happen if the cases already have those events and we try to navigate to them?

Reply:
The event is removed from history tab

Reply:
Added back after discussing offline